src/modules/visitor-space/components/AutocompleteFieldInput/AutocompleteFieldInput.module.scss

```diff
@@ -0,0 +1,12 @@
+@use "src/styles/abstracts" as *;
+
+$component: "c-autocomplete-field-input";
+
+.#{$component} {
+	width: 100%;
```

Comment:
I'm not a fan of defining this class in a different component variable. It just makes it more complicated for no reason.

Reply:
i agree
but it is the current coding style of the repo
if we decide we want to change it we need to agree with all devs that work on the project before we start changing the code